PowerPC64 targets using Little-endian fails to build

When packaging uutils coreutils 0.7.0 I discovered what appears to be a bug in the `print_terminal_size` function under `src/uu/stty/src/stty.rs` for ppc64 systems using little-endian. I have a patch ready to fix this.

## Problem

The external crate function `nix::sys::termios::cfgetospeed` returns a value of type `u32` for little-endian ppc64 systems where it is assumed to return an enum of `BaudRate`, leading to a compilation error. 

## Reproduce

Build uutils coreutils version 0.7.0 on a little-endian ppc64 system (this is a bit inaccessible to most).

## Fix

Assume the type of `speed` is `u32` on little-endian ppc64 targets.

## References

Build log for ppc64el on Ubuntu: https://launchpadlibrarian.net/850862195/buildlog_ubuntu-resolute-ppc64el.rust-coreutils_0.7.0-0ubuntu1~resolute2_BUILDING.txt.gz 

**Excerpt**:
```rust
error[E0308]: mismatched types
   --> src/uu/stty/src/stty.rs:674:40
    |
674 |             .find(|(_, rate)| *rate == speed)
    |                               -----    ^^^^^ expected `BaudRate`, found `u32`
    |                               |
    |                               expected because this is `BaudRate`
```

--- a/src/uu/stty/src/stty.rs
+++ b/src/uu/stty/src/stty.rs
@@ -631,26 +631,17 @@ fn print_terminal_size(
     #[cfg(not(target_os = "linux"))]
     let speed = nix::sys::termios::cfgetospeed(termios);
     #[cfg(target_os = "linux")]
-    #[cfg(all(
-        not(target_arch = "powerpc"),
-        not(target_arch = "powerpc64")
-    ))]
+    #[cfg(all(not(target_arch = "powerpc"), not(target_arch = "powerpc64")))]
     ioctl_read_bad!(tcgets2, TCGETS2, termios2);
     #[cfg(target_os = "linux")]
-    #[cfg(all(
-        not(target_arch = "powerpc"),
-        not(target_arch = "powerpc64")
-    ))]
+    #[cfg(all(not(target_arch = "powerpc"), not(target_arch = "powerpc64")))]
     let speed = {
         let mut t2 = unsafe { std::mem::zeroed::<termios2>() };
         unsafe { tcgets2(opts.file.as_raw_fd(), &raw mut t2)? };
         t2.c_ospeed
     };
     #[cfg(target_os = "linux")]
-    #[cfg(any(
-        target_arch = "powerpc",
-        target_arch = "powerpc64"
-    ))]
+    #[cfg(any(target_arch = "powerpc", target_arch = "powerpc64"))]
     let speed = nix::sys::termios::cfgetospeed(termios);
 
     let mut printer = WrappedPrinter::new(window_size);

